src/js/tooltip.js

```diff
@@ -138,9 +137,14 @@ class Tooltip {
 	 * Render the tooltip. Adds markup and attributes to this.tooltipEl in the DOM
 	*/
 	render() {
-		// make sure the tooltip is the next sibling of the target and (in the case of generated tooltip el)
-		// is attached to the DOM
-		if (this.targetNode && this.targetNode.nextSibling !== this.tooltipEl) {
+		// Make sure the tooltip is attached to the DOM
+		if (this.opts.appendToBody) {
+			// either appended directly into the body
+			if (!document.getElementById(this.opts.target + this.constructor.idSuffix)) {
```

Comment:
Sorry one more pedanic thing. Jake pointed out we might want `Tooltip.idSuffix` over `this.constructor.idSuffix`. 

If Tooltip is extended and `idSuffix` isn't defined on the extending class this will break.